wikimedia/wikipedia-ios
# Update Web donate urls

**Phabricator:** https://phabricator.wikimedia.org/T350919

### Notes
This PR updates the web donate urls:

From settings, it updates the `utm_source` value to `appmenu`, and adds `app_version`.
From a campaign modal, it adds `app_version`.

### Test Steps
1. On device, change device region to Japan. Change device date to May 31st.
2. Add line break [here](https://github.com/wikimedia/wikipedia-ios/blob/main/Wikipedia/Code/ViewController%2BURLHandling.swift#L15) in Xcode.
3. Fresh install on Staging app. On Explore feed, background, forground, then pull to refresh to fetch campaigns.
4. Visit an article on EN or JA Wikipedia. Campaign modal should appear. Tap Donate button. If needed choose Other payment method.
5. You should hit line break. Inspect the url, confirm `app_version` is appended to it.
6. Go to Settings. Tap Donate method. If needed choose Other payment method.
7. You should hit line break. Inspect the url, confirm `app_version` is appended to it, and `utm_source` = `appmenu`.
